routes/apiRouter.js

```diff
@@ -0,0 +1,51 @@
+import express from 'express'
+import { sharksList, catsList } from '../data/images.js'
+
+const apiRouter = express.Router()
+
+/**
+ * Return a random integer in the range [min, max) 
+ */
+const randomInt = (min, max) => {
+    return Math.floor(Math.random() * (max - min) + min)
+}
+
+/**
+ * Randomly shuffle an array in-place using the Fisher-Yares algorithm.
+ */
+const shuffle = (array) => {
+    for (let i = 0; i < array.length - 1; i++){
+        const swapIndex = randomInt(i, array.length - 1)
+
+        // swap i and swapIndex
+        const temp = array[swapIndex]
+        array[swapIndex] = array[i]
+        array[i] = temp
+    }
+    return array
+}
+
+apiRouter.get('/images', (req, res, next) => {
+    const content = req.query.content
```

Comment:
Can you explain why you chose to design your API the way that you did?

Reply:
I went back and forth a bit about whether to treat the each list of images as its own resource or to treat the entire list as one resource which could be filtered. This is where I'd normally do a sanity check with a coworker to hear what they thought about it.

I ended up designing it such that all the images are a single resource and then filtering by content as this is what one usually sees when delivering lists via an API. This was my balance between simplicity of design for this use case versus designing an API that would still make sense if more categories were added. However, I think it would also have made sense to treat each image list as a separate resource.

Interested to know what your thoughts are!